src/utils/chain.ts

```diff
@@ -13,18 +13,18 @@
  */
 
 import cli from 'cli-ux';
-import pgPromise from 'pg-promise';
```

Comment:
can you also remove the dependency in package.json? also check if we need to remove other unwanted code/libraries